app/main.ipynb

```diff
@@ -64,15 +115,37 @@
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
    "source": [
-    "# write your code here"
+    "product_sales = df['name'].value_counts()\n",
+    "top_10_products = product_sales.head(10)\n",
+    "total_sales = top_10_products.sum()\n",
+    "top_10_products_percentage = (top_10_products / total_sales) * 100\n",
+    "\n",
+    "plt.figure(figsize=(10, 10))\n",
+    "plt.pie(top_10_products_percentage, labels=top_10_products.index, autopct='%1.1f%%')\n",
+    "plt.title('Popularity of Top 10 Products by Sales Count')\n",
+    "plt.axis('equal')\n",
+    "plt.show()"
```

Comment:
this approach doesn't work because each product has quantity (number of orders), like your mates go to the restaurant and order soup for each person(e.g. 3 people), it's one order, thus 1 row, but quantity = 3, thus values_count will return you 1 instead of 3. Also quantity is mentioned in task description